frontend/src/components/RequestFormInputs.vue

```diff
@@ -212,21 +176,14 @@ const addToken = async () => {
 
 const addTokenButtonDisabled = computed(() => !ethereumProvider.signer || !selectedToken.value);
 
-const gasFeesTooltipText = `The fee amount is composed of three parts:
-  • the gas reimbursement fee
-  • the liquidity provider fee
-  • the Beamer service fee
-
-The gas reimbursement fee and the liquidity provider fee are paid out to the liquidity provider servicing the request, while the Beamer service fee stays with the contract and supports the Beamer platform development.
-
-Note that the fee is paid on top of the token amount being transferred,so that the token amount received on the target rollup is exactly the same as the token amount sent from the source rollup.`;
-
 const { provider, signer, chainId } = storeToRefs(ethereumProvider);
 
 const requestManagerAddress = computed(
   () => configuration.chains[chainId.value]?.requestManagerAddress,
 );
 
+const { CHAINS, TARGET_CHAINS, sourceChainId, switchChain } = useChainSelection(provider);
```

Comment:
As the `TARGET_CHAINS` (now `targetChains`) depend on the `sourceChainId` I think the naming is not too bad here. But I also get your point, I also think it is not the "prettiest" way. So, I can't come up with something better, I would leave it for now.
Also, I think the chain switching is really important here, because functionalities like the token balance depend on it. This means the provider, where we read values from, should be able to read from the source chain in order to provide such functionalities.

Reply:
> Also, I think the chain switching is really important here, because functionalities like the token balance depend on it. This means the provider, where we read values from, should be able to read from the source chain in order to provide such functionalities.

The point is that I would like to change that too. 😬 

Anyway. Nothing that holds back this PR.